secrets.go

```diff
@@ -181,9 +182,59 @@ func secretMounts(filePath, mountLabel, containerWorkingDir string) ([]rspec.Mou
 
 		mounts = append(mounts, m)
 	}
+
+	// Add FIPS mode secret
+	_, err := os.Stat("/etc/system-fips")
+	// The filePath check is to ensure that this secret is created after the
+	// DefaultMountsFile has been looked at, i.e builder.DefaultMountsFile
+	if err == nil && filePath != OverrideMountsFile {
```

Comment:
You want this to happen even if the OverrideMountsFile ispresent, I think

Reply:
This check is here because SecretMounts is called twice, once for the override mounts and the other for default causing the FIPS secret code to happen twice. So I put the check to make sure it just happens one time.